scalus-core/shared/src/main/scala/scalus/serialization/flat/package.scala

```diff
@@ -228,6 +228,29 @@ package object flat:
             val b = summon[Flat[B]].decode(decode)
             (a, b)
 
```

Comment:
Maybe let's generalize this to `Seq`? What do you think, @rssh?

Reply:
Yes, possible